tests/pipelines_components/automl/conftest.py

```
@@ -139,9 +141,10 @@ def automl_run_id(
         )
 
     yield run_id
-    delete_pipeline_run(
-        api_url=dspa_api_url,
-        headers=dspa_auth_headers,
-        run_id=run_id,
-        ca_bundle=dspa_ca_bundle_file,
-    )
+    if not os.getenv("SKIP_TEARDOWN", "").lower() in ("true", "1", "yes"):
```

Ruff (0.15.20)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)